README.md

```diff
@@ -1,2 +1,63 @@
 # Mergebot
-Mergebot is a GitHub Action that automates merging Dependabot PRs
+
+Mergebot is a GitHub Action that automates merging Dependabot PRs. It supports semantic versioning, enabling you to choose how big of an upgrade bump you allow, and automatically adds reviewers in case the PR requires special attention
+
+Even though it is designed with Dependabot in mind, Mergebot can also be used to automate other GitHub users' PRs. You simply need to override the default input for this
+
+# Dependabot config
+
+Below is a sample dependabot.yml. Notice the schedule is set to daily. This makes the process less conflict-prone. Mergebot can run with any Dependabot config
+
+    version: 2
+    updates:
+      - package-ecosystem: docker
+        directory: /
+        schedule:
+          interval: daily
+      - package-ecosystem: npm
+        directory: /
+        schedule:
+          interval: daily
+      - package-ecosystem: nuget
+        directory: /
+        schedule:
+          interval: daily
+
+# Mergebot config
+
+Below is a sample workflow that uses Mergebot. Notice we recommend checking whether the check_suite was triggered by Dependabot and it finished successfully. This is not necessary, as the action will check this also, but it saves a few seconds from the runtime
+
+Mergebot only supports the check_suite trigger, ensuring it only runs after all the other checks
+
+    name: Dependabot auto merge
+
+    on:
+      check_suite:
+        types:
+          - completed
+
+    jobs:
+      dependabot-auto-merge:
+        if: github.actor == 'dependabot[bot]' && github.event.check_suite.conclusion == 'success'
+        runs-on: ubuntu-latest
+        steps:
+          - uses: e-conomic/mergebot@v0.0.3
```

Comment:
Is the version required? If omitted will it use the latest version?

Reply:
It is required, yes

Reply:
Does it support the @latest handle? i.e. `e-conomic/mergebot@latest`

Reply:
Do we want to do that? We'll remove the option of having breaking changes

Reply:
That's true, but I am concerned of having to update tens of repos. And it effect not being able to cleanup old mergebot code, because there are repos out there which use an ancient version...

Reply:
We can actually constantly update a major version, until we need to do breaking changes

So we'd be able to say uses: e-conomic/mergebot@v1 and have it always use the latest v1. Makes sense?

Reply:
It does, but supporting v 1 always means keeping old code in the mergebot repo indefinitely. Do we want that?